### PR TITLE
doc: (IAC-441) Updated the CAS resources and Postgres sections

### PR DESCRIPTION
**Pull request by riragh**
# Changes:
Updated the CAS resources and Postgres sections in the README to reflect additional information.

### CODE REVIEW

File: docs/user/Multi-Tenancy.md [hunk 1]
```diff
@@ -4,14 +4,14 @@ SAS Viya supports a multi-tenant environment where multiple tenants can use the
...
 
 2. SAS Infrastructure Data Server. 
 
    SAS Viya requires either an internal PostgreSQL instance, which is the default option that is deployed automatically, or an external instance that you configure and maintain. Both the internal and external PostgreSQL options are supported for multi-tenancy. If you deploy with the default option, SAS configures and maintains the deployment for you. If you instead deploy an external PostgreSQL instance, you are responsible for configuring and maintaining it. For external PostgreSQL, see [Requirements for External PostgreSQL](https://go.documentation.sas.com/doc/en/itopscdc/default/itopssr/p05lfgkwib3zxbn1t6nyihexp12n.htm#p1wq8ouke3c6ixn1la636df9oa1u). Also for details see [PostgreSQL Requirements for a Multi-Tenant Deployment](https://go.documentation.sas.com/doc/en/itopscdc/default/itopssr/p05lfgkwib3zxbn1t6nyihexp12n.htm#p1r5u2f0yyiql5n11qb61lldcq1j).
 
+   **Note**: You need to size the total number of tenants that will onboarded to calculate max_connections correctly before deployment.
```
Comment (thpang) [inline docs/user/Multi-Tenancy.md:13]: I would emphasize this cannot be changed once it's been deployed. And if the customer needs to change, they would have to rebuild their cluster.

**Review by thpang — CHANGES_REQUESTED** [review]
Comment below

File: docs/user/Multi-Tenancy.md (hunk 1 above)
Comment (riragh) [reply docs/user/Multi-Tenancy.md]: Made the requested changes, along with rewording of required variables in the `Preparation` section.

**Review by sayeun — APPROVED** [review]
LGTM

**Review by thpang — APPROVED** [review]
LGTM

**Pull request merged**